src/main/scala/io/citrine/lolo/PredictionResult.scala

```diff
@@ -120,14 +119,16 @@ trait RegressionResult extends PredictionResult[Double] {
   def getQuantileMean(quantile: Double): Option[Seq[Double]] = None
 
   /**
-   * Get the estimated magnitude of the bias of each prediction, if possible
+   * **EXPERIMENTAL** Get the estimated magnitude of the bias of each prediction, if possible
```

Comment:
"magnitude" sounds like an absolute value (only the magnitude, not the sign), but the comment below says it's signed.